packages/components/src/components/data-list/mobile-description.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Text from '../text';
+
+const MobileDescription = ({ desc, desc_titles }) => {
+    if (desc.message.substring(0, 7) === 'Address') {
```

Comment:
we can't parse the message like this, it won't work for non english languages.

Reply:
I was looking for a way to differentiate the type of messages but the response form BE is the same, just a string. The only difference is the string contents. I'll try to look for a better way